src/assets/compendiums/demo-actors.db/adventurer.yaml

```diff
@@ -0,0 +1,1672 @@
+name: Demo Adventurer
+type: character
+img: systems/rqg/assets/images/items/hit-location.svg
+data:
+  characteristics:
+    strength:
+      value: 10
+      formula: 3d6
+    constitution:
+      value: 5
+      formula: 3d6
+    size:
+      value: 14
+      formula: 2d6+6
+    dexterity:
+      value: 13
+      formula: 3d6
+    intelligence:
+      value: 13
+      formula: 2d6+6
+    power:
+      value: 7
+      formula: 3d6
+      hasExperience: false
+    charisma:
+      value: 12
+      formula: 3d6
+  background:
+    species: Human
+    occupation: farmer
+    homeland: sartar
+    birthyear: null
+    age: 21
+    gender: Man
+    town: Demo Town
+    tribe: null
+    clan: null
+    reputation: 5
+    standardOfLiving: Free
+    baseIncome: null
+    ransom: null
+    ransomDetails: null
+    biography: <p>This is a place to put background info and links to Journal Entries about the adventurer.</p>
+  allies: <p>@Compendium[rqg.demo-actors.ro196XqFnrNvYB69]{Demo Spirit}</p>
+  attributes:
+    magicPoints:
+      value: 10
+    hitPoints:
+      value: 8
+    move:
+      currentLocomotion: walk
+      walk:
+        value: 8
+        carryingFactor: 1
+      swim:
+        value: 2
+        carryingFactor: 0.5
+      fly: {}
+    heroPoints: 0
+    health: healthy
+token:
+  name: Demo Adventurer
+  img: systems/rqg/assets/images/items/hit-location.svg
+  displayName: 0
+  actorLink: true
+  width: 1
+  height: 1
+  scale: 1
+  mirrorX: false
+  mirrorY: false
+  lockRotation: false
+  rotation: 0
+  alpha: 1
+  vision: true
+  dimSight: 30
+  brightSight: 0
+  dimLight: 0
+  brightLight: 0
+  sightAngle: 0
+  lightAngle: 0
+  lightAlpha: 0.25
+  lightAnimation:
+    speed: 5
+    intensity: 5
+  disposition: -1
+  displayBars: 0
+  bar1:
+    attribute: attributes.hitPoints.value
+  bar2:
+    attribute: attributes.magicPoints.value
+  randomImg: false
+items:
+  - _id: UQ8H7xAJIgpVG2Dk
+    name: Abdomen
+    type: hitLocation
+    img: systems/rqg/assets/images/items/hit-location.svg
+    data:
```

Comment:
I believe the hit locations should have rqid stuff.  Given how we're discussing changing rqid to flags and also that this is more about setting up the build pipeline than it is the content, I'm ok with leaving this for now.  Is there some way we can define say "Abdomen" in one place and reference that on all actors and in the hit locations compendium?  I suppose they'd need unique _id values?